web/src/styles/sections/login-page.scss

```diff
@@ -23,6 +23,15 @@
   p {
     color: variables.$base-dark;
   }
+  .need-help-text button {
+    margin-top: 0;
+  }
+
+  @media (max-width: variables.$sm) {
+    .help-text-container {
+      max-width: 16.25rem;
```

Comment:
16.25 * 16px (typical root element `font-size`) = 260. Ensures better wrapping of the text here